commands.go

```diff
@@ -694,3 +698,36 @@ func lsCMD(rest string, u *user) {
 func commandsCMD(_ string, u *user) {
 	u.room.broadcast("", "Commands  \n"+autogenCommands(cmds))
 }
+
+func autoloadCMD(arg string, u *user) {
+	if arg == "on" {
+		u.autoload = true
+	} else if arg == "off" {
+		u.autoload = false
+	} else if arg == "" {
+		var aload string
+		if u.autoload {
+			aload = "on"
+		} else {
+			aload = "off"
+		}
+		u.room.broadcast("", u.name+" has autoload "+aload)
+	} else {
+		u.room.broadcast("", "usage: autoload on|off")
+	}
+}
+func saveCMD(_ string, u *user) {
+	err := u.savePrefs()
+	if err != nil {
+		u.room.broadcast(devbot, "An error occured while saving, see the console for more info")
+		fmt.Fprintf(os.Stderr, "error while saving user data: %v\n", err)
+	}
+
+}
+func loadCMD(_ string, u *user) {
+	err := u.loadPrefs(false)
+	if err != nil {
+		u.room.broadcast(devbot, "An error occured while loading, see the console for more info")
+		fmt.Fprintf(os.Stderr, "error while loading user data: %v\n", err)
+	}
+}
```

Comment:
What would a user need these commands for? AFAICT, their prefs are already saved and loaded automatically when they join and leave.

Reply:
If they don't want auto save, or want to have a default

Reply:
Oh, is it to reset to saved preferences (undo changes?)